### PR TITLE
travis CI

### LINKED ISSUE #106: Add travis CI integration
Build PRs and master with Travis CI. For this ticket, simply building the app (running npm install without error etc.) is enough, tests, code style etc. to be done as part of other tickets.

### PR DESCRIPTION
**Pull request by DirtyF**
fix #106

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+language: node_js
+node_js:
+- 8
```
Comment (jenkoian) [inline .travis.yml]: Think we should build against latest stable and latest LTS https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Specifying-Node.js-versions

**Comment by jenkoian** [comment]
Thanks @DirtyF - a few bits of feedback:

* I think we should do this https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Caching-with-npm
* Should be using yarn, so think we should add this as well https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Caching-with-yarn

**Comment by jenkoian** [comment]
Now that https://github.com/jenkoian/hacktoberfest-checker/pull/114 has been merged, would be good to add this as well 👍

**Comment by DirtyF** [comment]
@jenkoian addressed your feedback 😄

**Comment by jenkoian** [comment]
Thanks @DirtyF !

**Pull request merged**